external/fv3viz/fv3viz/_plot_cube.py

```diff
@@ -159,16 +160,26 @@ def plot_cube(
         cmap=kwargs.get("cmap"),
         robust=cmap_percentiles_lim,
     )
-
-    _plot_func_short = partial(
-        _plot_cube_axes,
-        lat=mappable_ds.lat.values,
-        lon=mappable_ds.lon.values,
-        latb=mappable_ds.latb.values,
-        lonb=mappable_ds.lonb.values,
-        plotting_function=plotting_function,
-        **kwargs,
-    )
+    if gsrm_name == "fv3":
```

Comment:
Glad this approach of separating the axes' plotting function from rest of the data handing seems to have worked out OK